src-docs/src/views/header/header_example.js

```diff
@@ -299,23 +299,34 @@ export const HeaderExample = {
           <p>
             Most consumers need a header that does not scroll away with the page
             contents. You can apply this display by applying the property{' '}
-            <EuiCode language="ts">{'position="fixed"'}</EuiCode>. This will
-            also add a class of <EuiCode>.euiBody--headerIsFixed</EuiCode> to
-            the window body.
+            <EuiCode language="ts">{'position="fixed"'}</EuiCode>. Multiple
+            fixed headers will automatically stack underneath one another
+            without manual positioning required.
```

Comment:
I think splitting the last sentence into two concrete statements makes them more impactful. The first clearly states what the prop gives consumers automatically. The second reinforces they don't have to do anything else.

```suggestion
            <EuiCode language="ts">{'position="fixed"'}</EuiCode>. Multiple
            fixed headers will automatically stack underneath one another. 
            No manual positioning is required.
```